pegasus/src/social_metadata.rb

```diff
@@ -227,7 +232,17 @@ def get_social_metadata_for_page(request)
   elsif request.path == "/" && ["code.org", "hourofcode.com"].include?(request.site)
     page = request.site
   elsif request.path == "/learn" && request.site == "hourofcode.com"
```

Comment:
This would be cleaner as
```
elsif ["/learn", "/how-to", "/how-to/events", "/resources", "/events", "/faq"].include?(request.path) && request.site == "hourofcode.com"
```

As it stands, I'm having a hard time understanding why they're split up. 

Related question: what paths don't use social metadata?

Reply:
I know this is something we're trying to get out today so I'd be okay with doing this as a follow-up

Reply:
I did it like this to keep the pattern of the others and will make a ticket to clean this up before the November launch. 

I'll need to do some digging to see where the default is set for hourofcode.com since at first glance it doesn't look like it's in this file.

Reply:
Made a ticket! [ACQ-2446](https://codedotorg.atlassian.net/browse/ACQ-2446)